docs/namespaces/tag.rst

```diff
@@ -85,7 +85,90 @@ by the schema.
     0.000 30.000   "reggae" null
     ===== ======== ======== ==========
 
+tag_msd_tagtraum_cd1
+~~~~~~~~~~~~~~~~~~~~
+Genre classes from the `msd tagtraum cd1`_ dataset.
 
+    ===== ======== ================== ==========
+    time  duration value              confidence
+    ===== ======== ================== ==========
+    [sec] [sec]    string             --
+    ===== ======== ================== ==========
+
+The ``value`` field is constrained to one of 13 strings:
+
+    - ``reggae``
+    - ``pop/rock``
+    - ``rnb``
+    - ``jazz``
+    - ``vocal``
+    - ``new age``
+    - ``latin``
+    - ``rap``
+    - ``country``
+    - ``international``
+    - ``blues``
+    - ``electronic``
+    - ``folk``
+
+.. _msd tagtraum cd1: http://www.tagtraum.com/msd_genre_datasets.html
+
+By convention, one or two tags per track are possible in this namespace.
+The sum of the confidence values should equal ``1.0``.
```

Comment:
If confidence is intended to be numeric `[0, 1]`, can we at least schematize this?  Constraining the sum would be impossible, but we can at least force proper syntax.

The `tempo` namespace gives an example of this kind of constraint.  I suspect we'd want it to be optional, in which case you'd have to use `oneOf` encodings as in the optional fields for `Annotation`: https://github.com/marl/jams/blob/master/jams/schemata/jams_schema.json#L103-L106